src/html/index.html.template

```diff
@@ -51,6 +51,7 @@
       @media (prefers-color-scheme: dark) {
         html {
           background-color: #111111;
+          --primary-text-color: #EEEEEE !important;
```

Comment:
```suggestion
          --primary-text-color: #e1e1e1 !important;
```

Reply:
I don't think we should use `!important`, will this be overruled by the user theme later?

Reply:
Oh. It will. How could I reproduce this situation, so I could find a better fix?

Reply:
Run this code, set your OS mode to dark and set a theme?

Reply:
I mean reproduce the error loading/ connecting to HA.

Reply:
Comment this: <https://github.com/home-assistant/frontend/blob/dev/src/util/init-skeleton.ts#L4>

Reply:
Would a better way to be to add a `style` tag to https://github.com/home-assistant/frontend/blob/dev/src/layouts/ha-init-page.ts?

Reply:
Where is the `--primary-color` defined used in `ha-init-page`?